packages/clerk-js/src/utils/url.ts

```diff
@@ -174,6 +174,12 @@ export function toURL(url: string | URL): URL {
  * @returns {string} Returns the URL href without the origin
  */
 export function stripOrigin(url: URL | string): string {
+  // In non-browser environments `window.location.origin` might not be available
+  // if not polyfilled, so we can't construct a URL object with the `url` string
+  if (typeof window.location === 'undefined' && typeof url === 'string') {
+    return url;
+  }
```

Comment:
❓ should we instead handle this in `toURL()` where the direct `window.location.origin` usage is?

Reply:
That was my initial thought, but I think `stripOrigin` is better suited for making sure `origin` is accessible, and as it accepts both `URL` and `string`, essentially the code looks like this:
- if you somehow managed to create a `URL` object, I'll pass it down to `toURL`
- if you passed me a string, and I cannot safely detect `origin` to remove it, I'll just return the string

Happy to change this if you think moving it to `toURL` is easier to understand though.

Reply:
@bratsos I don't think this covers the case of passing a fully qualified URL as a string, e.g.

```
stripOrigin("https://clerk.com/blog") // -> /blog
```

In this case, I would still expect it to strip the origin. `toURL()` only uses `window.location.origin` as the fallback.

Reply:
Sure: https://github.com/clerk/javascript/pull/6737/commits/ff7748eb25e460e6e2326857d58e50e5f42b80bb